core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -850,22 +890,19 @@ class ConfigCommandTest extends ZooKeeperTestHarness with Logging {
   }
 
   @Test
-  def shouldAddBrokerQuotaConfig(): Unit = {
```

Comment:
We have a `shouldAddBrokerQuotaConfig` test with bootstrap-server?

Reply:
Yes, `shouldAddBrokerDynamicConfig` is testing it. I added one more quota config update to verify it.